packages/zosfiles/__tests__/__system__/api/methods/download/Download.system.test.ts

```diff
@@ -123,6 +123,45 @@ describe("Download Data Set", () => {
                 expect(fileContents).toEqual(data);
             });
 
+            it("should download a data set and create folders and file in original letter case", async () => {
```

Comment:
If we are still keeping two tests, at least we need to make the names different so that reports (or possible snapshots) don't present unexpected behavior.

Reply:
You are right, I should've expanded the file a bit more! 😋 
Thanks!